src/demo/initTemporaryAccount.ts

```
@@ -0,0 +1,27 @@
+import 'dotenv-defaults/config';
+
+import {Account} from '@wireapp/core';
+import {ClientType} from '@wireapp/api-client/src/client';
+import {APIClient} from '@wireapp/api-client';
+
+const {WIRE_EMAIL, WIRE_PASSWORD} = process.env;
+
+export async function initTemporaryAccount(): Promise<Account> {
+  const apiClient = new APIClient();
+
+  const account = new Account(apiClient);
+
+  const clientType = ClientType.TEMPORARY;
+
+  await account.login({
+    clientType,
+    email: WIRE_EMAIL,
+    password: WIRE_PASSWORD,
+  });
```

Maybe throw an error if `WIRE_EMAIL` or `WIRE_PASSWORD` is undefined?